extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -571,8 +571,13 @@ export async function canTargetConnectToStorageAccount(
 
 			break;
 		case MigrationTargetType.SQLVM:
-			// to-do: VM scenario -- get subnet by first checking underlying compute VM, then its network interface
-			return true;
+			const targetVmNetworkInterfaces = Array.from((await NetworkInterfaceModel.getVmNetworkInterfaces(account, subscription, (targetServer as SqlVMServer))).values());
```

Comment:
can getVmNetworkInterfaces return null and throw on .values() call?

Reply:
for this call particularly no - `getVmNetworkInterfaces` will always return something, defaulting to empty